pages/linux/uuidd.md

```diff
@@ -0,0 +1,16 @@
+# uuidd
+
+> Daemon for generating UUIDs.
+> More information: <https://manned.org/uuidd>.
+
+- Generate a random UUID:
+
+`uuidd --random`
+
+- Generate a bulk number of UUIDs:
+
+`uuidd --random --uuids {{number_of_uuids}}`
+
+- Generate a time based UUID:
+
+`uuidd --time`
```

Comment:
Does there not need to be a time parameter

Reply:
Not sure what you're getting at, @CleanMachine1.

Reply:
I can vouch it's not required at least, I've tried to find more information on this but didn't see anything, and the man pages make no mention of a value for the argument.

```log
seth@Rogeliina:~$ uuidd --time
7a03154c-b82b-11eb-8d10-8df892758bb2
seth@Rogeliina:~$ uuidd --time
7b1946fe-b82b-11eb-8d10-8df892758bb2
seth@Rogeliina:~$ uuidd --time
7ba80326-b82b-11eb-8d10-8df892758bb2
```

When testing it with a real values, the behavior doesn't change.

I've tried to peek again before making this comment, but can only assume based on information on UUID v1 that it generates one using the current timestamp, and that's all it supports.

> Version 1 concatenates the 48-bit MAC address of the "node" (that is, the computer generating the UUID), with a 60-bit timestamp, being the number of 100-nanosecond intervals since midnight 15 October 1582 Coordinated Universal Time (UTC), the date on which the Gregorian calendar was first adopted.
> \- https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_1_(date-time_and_MAC_address)

I could update this to specify that it uses the current timestamp and MAC address, though.

```diff
- - Generate a time based UUID:
+ - Generate a time based UUID, based on the time and MAC address of the system:
```

Reply:
To be honest I don't even know what uuid is and how to use it on linux

Need to look at the tldr page 😁